<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudAuthor</class>
 <widget class="QMainWindow" name="CrudAuthor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>671</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administración de autores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/book-flat.ico</normaloff>:/book-flat.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbInsert">
     <attribute name="title">
      <string>Agregar autor</string>
     </attribute>
     <widget class="QLabel" name="lbIdAuthor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>62</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Id autor</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbAuthor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>92</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(35, 54, 86);</string>
      </property>
      <property name="text">
       <string>Administración de autores</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneIdAuthor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneFirstName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>55</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbInsert">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pbInsert {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbInsert:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>Insertar autor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbExit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pbExit {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>Salir al menú principal</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbAuthor_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>120</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ap. paterno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneFatherLastName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>118</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>55</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbAuthor_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ap. materno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneMotherLastName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>55</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbQuery">
     <attribute name="title">
      <string>Buscar autor</string>
     </attribute>
     <widget class="QLabel" name="lbSearch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Busqueda:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneCritery">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>12</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboCritery">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>12</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvQuery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>621</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbUpdate">
     <attribute name="title">
      <string>Modificar autor</string>
     </attribute>
     <widget class="QGroupBox" name="gbIdCareer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lneIdAuthorQuery">
       <property name="geometry">
        <rect>
         <x>77</x>
         <y>13</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="lbIdAuthorQuery">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>15</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id autor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSearch">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbSearch {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbSearch:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbAuthor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>621</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pbExit2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>80</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbExit2 {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit2:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Salir al menú principal</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbUpdate">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbUpdate {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbUpdate:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Actualizar autor</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbAuthor_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ap. materno</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneFatherLastNameUpdate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>68</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>55</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneMotherLastNameUpdate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>55</number>
       </property>
      </widget>
      <widget class="QLabel" name="lbAuthorUpdate">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>70</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ap. paterno</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbAuthor_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>42</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneFirstNameUpdate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>55</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>-5</y>
      <width>691</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 91, 118);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lneIdAuthor</tabstop>
  <tabstop>lneFirstName</tabstop>
  <tabstop>pbInsert</tabstop>
  <tabstop>pbExit</tabstop>
  <tabstop>lneCritery</tabstop>
  <tabstop>cboCritery</tabstop>
  <tabstop>lneIdAuthorQuery</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pbSearch</tabstop>
  <tabstop>pbExit2</tabstop>
  <tabstop>pbUpdate</tabstop>
  <tabstop>tvQuery</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>CrudAuthor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneIdAuthor</sender>
   <signal>returnPressed()</signal>
   <receiver>lneFirstName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneFirstName</sender>
   <signal>returnPressed()</signal>
   <receiver>lneFatherLastName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneFatherLastName</sender>
   <signal>returnPressed()</signal>
   <receiver>lneMotherLastName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbExit2</sender>
   <signal>clicked()</signal>
   <receiver>CrudAuthor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
